src/hyp3_autorift/process.py

```diff
@@ -582,6 +583,11 @@ def main():
         default=True,
         help='Use static topographic correction files for ISCE3 processing if available (Sentinel-1 only).',
     )
+    parser.add_argument(
+        '--frame-id',
+        default='',
```

Comment:
```suggestion
        type=nullable_string,
        default=None,
```

HyP3 will need to always pass something here, so it'll end up being the string `None` or the value provided.